app/models/session.rb

```diff
@@ -286,6 +286,19 @@ def open_for_consent?
     close_consent_at&.today? || close_consent_at&.future? || false
   end
 
+  def next_reminder_date
+    reminders = send_consent_reminders_at
+    return nil unless reminders
+    reminders.first
+  end
```

Comment:
I think something like this would work?
```suggestion
  def next_reminder_date = send_consent_reminders_at&.first
```

Reply:
Excellent suggestion